src/components/CarouselHeader.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { render } from '@testing-library/react-native'
+import CarouselHeader from './CarouselHeader'
+
+test('Should render using props', () => {
+  const { getByText } = render(
+    <CarouselHeader
+      title="Test Title"
+      subheading="Test subheading"
+      icon={require('../images/icons/activity/hiking.png')}
```

Comment:
**question (non-blocking)**: Can this be an import at the top of the file?

Reply:
Probably, this is just the way the docs show it being done.

I didn't focus too much time on it as it's likely going to change.

Reply:
Oh whoops, this is the test file. Yeah, doing it once at the top of the file probably makes more sense for when/if there are more than one test.